server/postgresApp.js

```diff
@@ -0,0 +1,163 @@
+const express = require('express');
+const app = express();
+const pgres = require('../PostgreSQL/index.js');
+const path = require('path');
+
+
+app.use(express.json())
+app.use('/products', express.static('./client/dist'))
+app.use('/', express.static('./client/dist'))
+
+pgres.connect();
+
+
+app.get('/api/productOptions/products/:id', (req, res) => {
+  let builtResponse;
+  let id = req.params.id;
+  let getItemsQuery = `SELECT * FROM items WHERE id = ${id}`;
+  let getReviewsQuery = `SELECT * FROM reviews WHERE prodId = ${id}`;
+
+  // ideally this would be a single query that joins the reviews result as a property of the items table, but I have to move ahead for now
+  pgres.query(getItemsQuery)
+    .then(result => {
+      builtResponse = result.rows[0];
+      builtResponse.price = {originalPrice: builtResponse.originalPrice, salePrice: builtResponse.salePrice};
+      delete builtResponse.originalPrice;
+      delete builtResponse.salePrice;
```

Comment:
not totally sure what's happening here--> reason for deletes? No need to answer here since I'm sure it works and there's plenty to do, just curious.

Reply:
I can answer! The reason is that originalPrice and salePrice are separate columns in the database, but the client expects them both within a price object. So I quickly recreate that object and get rid of the originalPrice and salePrice properties to 'trim the fat' on the response to the client. Probably not necessary since a user never actually sees the response, though!